src/test/java/com/microsoftopentechnologies/windowsazurestorage/helper/CredentialRenameTest.java

```diff
@@ -5,80 +5,29 @@
 import com.cloudbees.plugins.credentials.domains.Domain;
 import hudson.model.Item;
 import hudson.security.ACL;
-import jenkins.model.Jenkins;
-import org.apache.commons.io.FileUtils;
-import org.junit.Before;
-import org.junit.ClassRule;
+import java.util.Collections;
+import java.util.List;
 import org.junit.Rule;
 import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
 import org.jvnet.hudson.test.JenkinsRule;
-
-import java.io.File;
-import java.util.Collections;
-import java.util.List;
+import org.jvnet.hudson.test.recipes.LocalData;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 
 public class CredentialRenameTest {
-    public static File input = new File("src/test/resources/com/microsoftopentechnologies/helper/correctFormatOldConfig.xml");
-    private Jenkins jenkinsInstance;
-    private static final String correctConfigContent =
```

Comment:
on new credentials it seems like the credential.xml was cached in memory and credentials weren't being loaded by just overwriting it on the file system